pl_bolts/datasets/mnist_dataset.py

```diff
@@ -13,36 +14,21 @@
 else:  # pragma: no cover
     warn_missing_pkg("PIL", pypi_name="Pillow")
 
-# TODO(akihironitta): This is needed to avoid 503 error when downloading MNIST dataset
-# from http://yann.lecun.com/exdb/mnist/ and can be removed after `torchvision==0.9.1`.
-# See https://github.com/pytorch/vision/issues/3549 for details.
-if _TORCHVISION_AVAILABLE and _TORCHVISION_LESS_THAN_0_9_1:
-    MNIST.resources = [
-        (
-            "https://ossci-datasets.s3.amazonaws.com/mnist/train-images-idx3-ubyte.gz",
-            "f68b3c2dcbeaaa9fbdd348bbdeb94873",
-        ),
-        (
-            "https://ossci-datasets.s3.amazonaws.com/mnist/train-labels-idx1-ubyte.gz",
-            "d53e105ee54ea40749a09fcbcd1e9432",
-        ),
-        (
-            "https://ossci-datasets.s3.amazonaws.com/mnist/t10k-images-idx3-ubyte.gz",
-            "9fb629c4189551a2d022fa330f9573f3",
-        ),
-        (
-            "https://ossci-datasets.s3.amazonaws.com/mnist/t10k-labels-idx1-ubyte.gz",
-            "ec29112dd5afa0611ce80d1b7f02629c",
-        ),
-    ]
-
 
-@under_review()
 class BinaryMNIST(MNIST):
-    def __getitem__(self, idx):
+    """Binarized MNIST Dataset.
+
+    MNIST dataset binarized using a thresholding operation. Threshold is set to 127. Note that the images are binarized
+    prior to the application of any transforms.
+    """
+
+    threshold = 127.0
```

Comment:
Can this threshold be a part of the `__init__` call already?

Reply:
Added threshold as an optional argument with default value as 127.